pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -114,6 +116,29 @@ pub struct IndexPart {
     /// The timestamp when the timeline was marked invisible in synthetic size calculations.
     #[serde(skip_serializing_if = "Option::is_none", default)]
     pub(crate) marked_invisible_at: Option<NaiveDateTime>,
+
+    /// The encryption key used to encrypt the timeline layer files.
+    #[serde(skip_serializing_if = "Option::is_none", default)]
+    pub(crate) keys: Option<Vec<EncryptionKey>>,
```

Comment:
I wonder if one can avoid the `Option` here somehow and instead deserialize to the empty vec if not present

Reply:
https://github.com/neondatabase/neon/pull/11573/commits/aba9eb3944c1e90f01ae05f6c0e0fcff49984e99